front/src/applications/stdcm/types.ts

```diff
@@ -28,6 +28,7 @@ export type StdcmSuccessResponse = Omit<
 
 export type SimulationReportSheetProps = {
   stdcmData: StdcmSuccessResponse;
+  stdcmSimulation: StdcmSimulation;
```

Comment:
Same here we could only the consist datas

```suggestion
  stdcmSimulationConsist: StdcmSimulation['inputs']['consist'];
```